blockchain/blockchain.go

```diff
@@ -247,7 +251,8 @@ func NewBlockChain(db database.DBManager, cacheConfig *CacheConfig, chainConfig
 		cacheConfig:        cacheConfig,
 		db:                 db,
 		triegc:             prque.New(),
-		chBlock:            make(chan gcBlock, 1000),
+		chBlock:            make(chan gcBlock, 2048), // downloader.maxResultsProcess
+		chPrune:            make(chan uint64, 2048),  // downloader.maxResultsProcess
```

Comment:
Are these channels related to downloader?

Reply:
@jeongkyun-oh See https://github.com/klaytn/klaytn/pull/1871#discussion_r1272376894